internal/util/version/version.go

```diff
@@ -73,7 +73,7 @@ func init() {
 		switch s.Key {
 		case "vcs.revision":
 			if s.Value != info.Commit {
-				panic(fmt.Sprintf("commit.txt value %q != vcs.revision value %q", info.Commit, s.Value))
+				panic(fmt.Sprintf("commit.txt value %q != vcs.revision value %q\nPlease run `bin/task gen-version`", info.Commit, s.Value))
```

Comment:
That looks good, but I think we should add `gen-version` task as a dependency for tasks where we need it

Reply:
I've added `gen-version` to `run` task but it's also happen when you edit and run right from editor. It confuses from time to time.